web/src/main/java/com/epam/rft/atsy/web/controllers/FileUploadController.java

```diff
@@ -63,7 +65,7 @@ public ModelAndView uploadFile(@PathVariable("candidateId") Long candidateId,
     try {
       CandidateDTO candidateDTO = candidateService.getCandidate(candidateId);
       String candidateCVFilename = candidateDTO.getCvFilename();
-      if (candidateCVFilename != null && !candidateCVFilename.isEmpty()) {
+      if (candidateCVFilename != null && StringUtils.isNotEmpty(candidateCVFilename)) {
```

Comment:
StringUtils.isNotEmpty method can handle null value, if the parameter is null, then a method will return false value. Check the documentation: https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/StringUtils.html#isEmpty%28java.lang.CharSequence%29